### PR TITLE
New solution

### PR DESCRIPTION
**Pull request by SophieDeBenedetto**
needs :eyes: 
@AnnJohn @dfenjves @ipc103 @deniznida @aviflombaum

### CODE REVIEW

**Comment by AnnJohn** [comment]
Assigned to @dfenjves for review

Deadline: 9/16/15

File: README.md [hunk 1]
```diff
@@ -1,26 +1,324 @@
...
+2. Learn how to use sessions to authorize, i.e. log in and log out users of a web application. 
 
-# Sessions and User Authorization 
+## User Authorization: Using Sessions 
 
-## Objectives
+### Logging In
 
-1. Build a Sinatra app with a Users model so that users can sign up for and sign in to your web app.
+What does it mean for a user to "log in"? The action of logging in the simple the action of storing a user's ID in the `session` hash. Here's a basic user log in flow:
```
Comment (dfenjves) [inline README.md]: This needs to be rephrased - maybe there's a typo?

File: README.md [hunk 2]
```diff
@@ -1,26 +1,324 @@
...
+
+### Part I: Models and Migrations
+
+Our `User` model has a few attributes. A user has a name, email and password. 
+
+#### Step 1: Migration
+
+Write a migration that creates a `Users` table with name, email and password. Run `rake db:migrate` and then run your test suite. 
+
+You'll see that you're passing a number of tests, including tests like these:
```
Comment (dfenjves) [inline README.md]: These tests should be failing before adding validations, no?

Comment (dfenjves) [inline README.md]: These tests aren't passing at this point. Getting the error:
```
  1) User validations is invalid without a name
     Failure/Error: expect(User.create(name: nil, email: "email@email.com", password: "password", password_confirmation: "password")).to_not be_valid
     ActiveRecord::UnknownAttributeError:
       unknown attribute 'password_confirmation' for User.
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,324 @@
...
+
+    erb :'/registrations/signup'
+end
+```
+
+* Navigate to `localhost:9383/registrations/signup`. You should see a page that says `"sign up below:"`. Let's make a sign up form! 
+
+* Open up `app/views/registrations/signup.erb`. Our signup form needs a field for name, email and password. It needs to `POST` data to the `'/registrations'` path, so your form action should be `/registrations'` and your form method should be `POST`. 
+
+* Once you've written your form, go ahead and place a `binding.pry` inside the `post '/registrations` route in `app/controllers/registrations_controller.rb`. Then, fill out the form in your browser and hit the `"Sign Up"` button.
```
Comment (dfenjves) [inline README.md]: Can we use `puts params` instead? when using pry in Nitrous with Sinatra things often crash.

Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 2 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

File: README.md (hunk 1 above)
Comment (SophieDeBenedetto) [inline README.md]: fixed.

**Comment by dfenjves** [comment]
Good to merge

**Pull request merged**